code/datums/ruins/space.dm

```diff
@@ -182,3 +182,71 @@
 	name = "Crashed Wizard Shuttle"
 	description = "A shuttle of the Wizard Federation, sent out to crush some wandless scum. Unfortunately, the pilot suffered a magic-related accident and the shuttle crashed into a nearby asteroid."
 	cost = 2
+
+/datum/map_template/ruin/space/abandonedtele
+	id = "abandonedtele"
+	suffix = "abandonedtele.dmm"
+	name = "Abandoned Teleporter"
+	description = "An old teleporter, seemingly part of what used to be a larger satellite."
+	allow_duplicates = FALSE // This has its own area with an APC, and 2 APCs for the same area breaks badly
+
+/datum/map_template/ruin/space/blowntcommsat
+	id = "blowntcommsat"
+	suffix = "blowntcommsat.dmm"
+	name = "Blown-out Telecommunications Satellite"
+	description = "The remains of an old telecommunications satellite once utilised by NanoTrasen. It lays derelict, with quite a few pieces missing."
+	cost = 5 // This is a chonky boy
+	allow_duplicates = FALSE // This has its own area with an APC, and 2 APCs for the same area breaks badly. Its also huge.
+
+/datum/map_template/ruin/space/clownmime
+	id = "clownmime"
+	suffix = "clownmime.dmm"
+	name = "Clown & Mime Mineral Deposits"
+	description = "A crash site of two opposing factions, both trying to complete mining trips for their own valuable minerals. While all the crew have long perished, the minerals are likely intact."
+
```

Comment:
allow_duplicates should probably default to FALSE for all ruins, and only be enabled for ruins which are nothing but a small collection of rocks in space. Anything with any sort of detail (even a small crashed ship) should probably not allow duplicates.

Reply:
I set most of these to `FALSE` because of the whole area thing, but I dont know if its worth defaulting this to `FALSE` and setting all the others to `TRUE`